### PR TITLE
Remove disabled packages

### PR DESCRIPTION
**Pull request by sonicdoe**
Removes all currently disabled packages from `test.json`. See https://github.com/standard/standard-packages/pull/32#issuecomment-530173455.

### CODE REVIEW

File: test.json [hunk 1]
```diff
@@ -1,145 +1,47 @@
 [
-  {
-    "name": "fs-extra",
-    "repo": "https://github.com/jprichardson/node-fs-extra",
-    "description": "fs-extra contains methods that aren't included in the vanilla Node.js fs package. Such as mkdir -p, cp -r, and rm -rf.",
-    "dependents": 3279,
-    "disable": true
```
Comment (feross) [inline test.json]: This package is on standard 14.1.0 and it is now passing. Let's keep it in the test suite and not mark it as disabled anymore.

Pro-tip: You can run `npm run test -- --disabled` to run the test suite against repos marked as disabled. If any of them pass, they can be re-enabled.

File: test.json [hunk 2]
```diff
@@ -1,145 +1,47 @@
...
     "repo": "https://github.com/blakeembrey/co-mocha",
     "description": "Enable support for generators in Mocha tests",
     "dependents": 388
   },
-  {
-    "name": "dotenv",
-    "repo": "https://github.com/motdotla/dotenv",
-    "description": "Loads environment variables from .env file",
-    "dependents": 377,
-    "disable": true
```
Comment (feross) [inline test.json]: Let's keep this one. It now passes, plus they're on standard 13 and they seem to be keeping up to date (14 only came out a month ago).

File: test.json [hunk 3]
```diff
@@ -1,145 +1,47 @@
...
     "repo": "https://github.com/wavded/babel-tape-runner",
     "description": "Babel + Tape for running your ES Next tests",
     "dependents": 233
   },
-  {
-    "name": "merge2",
-    "repo": "https://github.com/teambition/merge2",
-    "description": "Merge multiple streams into one stream in sequence or parallel.",
-    "dependents": 214,
-    "disable": true
```
Comment (feross) [inline test.json]: This package is on standard 14.2.0. Leave it.

File: test.json [hunk 4]
```diff
@@ -198,54 +79,12 @@
...
     "repo": "https://github.com/feross/buffer",
     "description": "Node.js Buffer API, for the browser",
     "dependents": 75
   },
-  {
-    "name": "tar-stream",
-    "repo": "https://github.com/mafintosh/tar-stream",
-    "description": "tar-stream is a streaming tar parser and generator and nothing else. It is streams2 and operates purely using streams which means you can easily extract/parse tarballs without ever hitting the file system.",
-    "dependents": 73,
-    "disable": true
```
Comment (feross) [inline test.json]: Leave this. It's passing now.

File: test.json [hunk 5]
```diff
@@ -258,108 +97,24 @@
...
-    "description": "A simple Browserify transform for adding required styles to the browser.",
-    "dependents": 60,
-    "disable": true
-  },
-  {
-    "name": "david",
-    "repo": "https://github.com/alanshaw/david",
-    "description": "Node.js module that tells you when your project npm dependencies are out of date.",
-    "dependents": 60,
-    "disable": true
```
Comment (feross) [inline test.json]: Leave this. It's passing now.

File: test.json [hunk 6]
```diff
@@ -758,20 +260,6 @@
...
-    "description": "jsreport extension adding API and studio",
-    "dependents": 14,
-    "disable": true
-  },
-  {
-    "name": "humanize-duration",
-    "repo": "https://github.com/EvanHahn/HumanizeDuration.js",
-    "description": "Convert millisecond durations to English and many other languages.",
-    "dependents": 14,
-    "disable": true
```
Comment (feross) [inline test.json]: Leave this. Passing now.

File: test.json [hunk 7]
```diff
@@ -796,27 +284,6 @@
...
-    "description": "spawn processes conveniently in tape tests and match against stdout/stderr streaming output",
-    "dependents": 13,
-    "disable": true
-  },
-  {
-    "name": "markdown-pdf",
-    "repo": "https://github.com/alanshaw/markdown-pdf",
-    "description": "Markdown to PDF converter",
-    "dependents": 13,
-    "disable": true
```
Comment (feross) [inline test.json]: Leave this. Passing now.

**Review by feross — CHANGES_REQUESTED** [review]
This generally looks good, but I'd like to keep a few of the repos you removed since they are currently passing.

**Comment by sonicdoe** [comment]
I’ve rebased onto `master` now that #36 is merged. This also addresses all requested changes.

**Review by feross — APPROVED** [review]
LGTM, thanks for all these updates :)

**Pull request merged**